### PR TITLE
chore(web): html markup validity

### PR DESCRIPTION
**Pull request by stjanilofts**
A number of mostly minor changes to fix the html markup validity. Fixes most of the errors/warnings listed here https://validator.w3.org/nu/?doc=https%3A%2F%2Fisland.is%2F

Adds "As" prop from Reakit props to Button which removes "type" from the button when used. Can still be used with text button variant. This is so we can use Button inside anchor elements.

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by oddsson — APPROVED** [review]

File: apps/web/components/FrontpageSlider/FrontpageSlider.treat.ts [hunk 1]
```diff
@@ -62,6 +62,7 @@ export const textItemVisible = style({
 })
 
 export const tabBullet = style({
+  display: 'inline-block',
```
Comment (kksteini) [inline apps/web/components/FrontpageSlider/FrontpageSlider.treat.ts:65]: Was this implicit before?
Does this rule change behaviour in any way?

Comment (stjanilofts) [reply apps/web/components/FrontpageSlider/FrontpageSlider.treat.ts:65]: The element using the class was a div but changed to span and needed to be block/inline-block

File: apps/web/screens/Home.tsx [hunk 2]
```diff
@@ -42,6 +43,7 @@ interface HomeProps {
 const Home: Screen<HomeProps> = ({ categories, news, page }) => {
   const namespace = JSON.parse(page.namespace.fields)
   const { activeLocale, t } = useI18n()
+  const Router = useRouter()
```
Comment (kksteini) [inline apps/web/screens/Home.tsx:46]: Do you need to import and use the Router here?

Comment (stjanilofts) [reply apps/web/screens/Home.tsx:46]: Ah, not used. I will remove.

File: apps/web/components/FrontpageSlider/FrontpageSlider.treat.ts (hunk 1 above)
Comment (kksteini) [reply apps/web/components/FrontpageSlider/FrontpageSlider.treat.ts:65]: :+1:

File: apps/web/screens/Home.tsx (hunk 2 above)
Comment (kksteini) [reply apps/web/screens/Home.tsx]: :+1:

**Review by kksteini — APPROVED** [review]
LGTM

**Pull request merged**